### ISSUE TITLE
Notification when download completes

### ISSUE THREAD

**Issue by noplanman** [Lemutar/Synoloader#35]
Would be cool to get a notification when a download completes.

This may cause a problem with #22 though, because we might not even be connected to the DS.

**Comment by Lemutar** [comment]
Hi,

I'm planing to move synoloader to the SDK frame-work and have already
implemented some functionality.
Are you interested to participate in this project to move Synoloader form
XUL to firefox SDK ?

Cheers

2016-02-12 3:55 GMT+01:00 Armando Lüscher <notifications@github.com>:

**Comment by noplanman** [comment]
Yes! At the moment I don't have time though, but after May it looks a lot better for me 👍
